### DISCUSSION TITLE
Can staleTimes provide a configuration for each routing page? I don't want to configure it globally.

### DISCUSSION THREAD

**Discussion by liqingitt** [vercel/next.js#68532] [Ideas]
### Goals

1.Provides a "staleTimes"-like configuration for each route to control the cache time of each client route

### Background

I need to ensure that every time I open this page, the data obtained is the latest, and I don’t want to put the initial request in the client component; I can only configure the "staleTimes" of "experimental" in "next-config" to 0 to achieve this function, but this will cause the cache of other routes to fail.

### Proposal

Just like the "dynamic" configuration

**Comment by hubertoschusch** [comment]
Huge bump
